pkg/report/fuchsia.go

```diff
@@ -53,23 +61,64 @@ func ctorFuchsia(cfg *config) (reporterImpl, []string, error) {
 }
 
 func (ctx *fuchsia) ContainsCrash(output []byte) bool {
-	return containsCrash(output, zirconOopses, ctx.ignores)
+	return containsCrash(output, fuchsiaOopses, ctx.ignores)
 }
 
 func (ctx *fuchsia) Parse(output []byte) *Report {
 	// We symbolize here because zircon output does not contain even function names.
 	symbolized := ctx.symbolize(output)
-	rep := simpleLineParser(symbolized, zirconOopses, zirconStackParams, ctx.ignores)
+	rep := simpleLineParser(symbolized, fuchsiaOopses, fuchsiaStackParams, ctx.ignores)
 	if rep == nil {
 		return nil
 	}
 	rep.Output = output
 	if report := ctx.shortenReport(rep.Report); len(report) != 0 {
 		rep.Report = report
 	}
+	if strings.HasPrefix(rep.Title, "starnix kernel panic") {
+		if report := ctx.shortenStarnixPanicReport(rep.Report, 5, 20); len(report) != 0 {
+			rep.Report = report
+		}
+	}
 	return rep
 }
 
+// Captures lines that match one of `starnixFramePatterns`, plus some surrounding lines that may
+// or may not be interesting.
+//
+// Captures up to `maxUnrelatedLines` of consecutive lines that do not start with the usual starnix
+// log prefix `starnixLinePrefix` before suppressing unrelated lines. These lines are often
+// syzkaller log lines, but are sometimes continuations of newline-containing logs from starnix.
+//
+// Captures up to `maxNonFrameLines` of consecutive starnix log lines that do not match one of
```

Comment:
Is this meant to be `maxUnmatchedLines`?

Reply:
Yes, thanks! Fixed.